### ISSUE TITLE
Haddock for Estimate trails off

### ISSUE THREAD

**Issue by RyanGlScott** [bos/statistics#102]
See [here](https://github.com/bos/statistics/blob/465a4121269cab4b800422fd0042373f8e9eaaf5/Statistics/Types.hs#L267-L272):

```haskell
data Estimate e a = Estimate
   { estPoint           :: !a
     -- ^ Point estimate.
   , estError           :: !(e a)
     -- ^ Confidence error estimate. It's parametrized by
   } deriving (Eq, Read, Show, Typeable, Data, Generic)
```

The documentation for `estError` cuts off at "It's parametrized by", making it hard to tell what `e` and `a` are.

**Comment by RyanGlScott** [comment]
Oops, this looks like a duplicate of #101.

**Comment by Shimuuar** [comment]
Thank you. I'll fix on Friday. 

Not quite duplicate so I'm reopening it

**Comment by Shimuuar** [comment]
Fixed in e74a8a73a49fb1d42eaa57d4608894471d33e9a3
